Maui.RevenueCat.InAppBilling/Platforms/Android/Extensions/EntitlementInfosExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Com.Revenuecat.Purchases;
+using Maui.RevenueCat.InAppBilling.Extensions;
+using Maui.RevenueCat.InAppBilling.Models;
+
+namespace Maui.RevenueCat.InAppBilling.Platforms.Android.Extensions;
+internal static class EntitlementInfosExtensions
+{
```

Comment:
to really reflect the inner type in the name probably replace `EntitlementInfosExtensions` by `EntitlementInfoDtosExtensions` ? same for iOS

Reply:
isn't the naming convention that you are using to call it <target object type>Extensions.cs...  In which case I did get one wrong, but it would be Android = EntitlementInfosExtensions.cs and ios = RCEntitlementInfosExtensions.cs  .  I made that change.  Let me know if I'm not getting it.

Reply:
Correct, it always ends with `Extensions.cs` suffix, because there could be more than 1 extension method.

> Android = EntitlementInfosExtensions.cs 
> iOS = RCEntitlementInfosExtensions.cs 

yes, these 2 should be now correct